src/pages/manage/productions/create.js

```diff
@@ -1,28 +1,314 @@
-import { Cookies } from 'react-cookie';
-import { Title } from '../../../components/publiq-ui/Title';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { Text } from '../../../components/publiq-ui/Text';
+import { Page } from '../../../components/publiq-ui/Page';
+import { TypeaheadWithLabel } from '../../../components/publiq-ui/TypeaheadWithLabel';
+import { getInlineProps, Inline } from '../../../components/publiq-ui/Inline';
+import { Button, ButtonVariants } from '../../../components/publiq-ui/Button';
+import { Event } from '../../../components/manage/productions/create/Event';
+import { getApplicationServerSideProps } from '../../../utils/getApplicationServerSideProps';
+import { parseEventId } from '../../../utils/parseEventId';
+import { useState, useMemo, useEffect } from 'react';
+import {
+  useGetProductions,
+  useGetSuggestedEvents,
+  useSkipSuggestedEvents,
+  useCreateWithEvents,
+  useMergeProductions,
+  useAddEventsByIds,
+} from '../../../hooks/api/productions';
+import { getStackProps, Stack } from '../../../components/publiq-ui/Stack';
+import { RadioButtonGroup } from '../../../components/publiq-ui/RadioButtonGroup';
+import { getBoxProps } from '../../../components/publiq-ui/Box';
+import { debounce } from 'lodash';
+import { useQueryClient } from 'react-query';
+import { QueryStatus } from '../../../hooks/api/authenticated-query';
+import { Spinner } from '../../../components/publiq-ui/Spinner';
+
+const ZeroOrOneProduction = ({
+  productionName,
+  suggestedProductions,
+  onInput,
+  onSelection,
+  ...props
+}) => {
+  const { t } = useTranslation();
+  return (
+    <TypeaheadWithLabel
+      id="typeahead-productionname"
+      options={suggestedProductions}
+      labelKey={(production) => production.name}
+      disabled={!!productionName}
+      placeholder={productionName}
+      maxWidth="43rem"
+      label={t('productions.create.production_name')}
+      emptyLabel={t('productions.create.no_productions')}
+      onInput={onInput}
+      onSelection={onSelection}
+      {...getBoxProps(props)}
+    />
+  );
+};
+
+ZeroOrOneProduction.propTypes = {
+  productionName: PropTypes.string,
+  suggestedProductions: PropTypes.array,
+  onInput: PropTypes.func,
+  onSelection: PropTypes.func,
+  className: PropTypes.string,
+};
+
+ZeroOrOneProduction.defaultProps = {
+  suggestedProductions: [],
+};
+
+const TwoProductions = ({ productionNames, onChange, ...props }) => {
+  const { t } = useTranslation();
+  return (
+    <RadioButtonGroup
+      name="production-names"
+      items={productionNames}
+      groupLabel={t('productions.create.production_name')}
+      onChange={onChange}
+      {...getStackProps(props)}
+    />
+  );
+};
+
+TwoProductions.propTypes = {
+  productionNames: PropTypes.array,
+  onChange: PropTypes.func,
+};
+
+TwoProductions.defaultProps = {
+  productionNames: [],
+};
```

Comment:
I think splitting these components up is creating more mental overhead than its actually helping us.